### DISCUSSION TITLE
ERORRS FAILED 2 WHICH CAUSE CAN'T RUN THE WEB INTERFACE.

### DISCUSSION THREAD

**Discussion by hsnroses** [Security-Onion-Solutions/securityonion#11680] [2.4]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8 cores

### RAM

16

### Storage for /

200 gb

### Storage for /nsm

200gb

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Why everytime i try to install securityonion 2 have a problem in installation process . when i run the log errors there are 2 erros display like picture below. 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/139231906/9b09fd1c-5971-4dac-8000-6aaa4497d2ee)

then when i run cat /opt/so/log/ it display like this 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/139231906/18c63c49-847a-4220-8c37-7ad10c893944)

i try a lot things to solve this error because my supervisor want me to present this securityonion 2 platform to the boss.Please help me thank youuu

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
> ERORRS FAILED 2 WHICH CAUSE CAN'T RUN THE WEB INTERFACE.

First, please edit the title of this discussion and remove the ALL UPPER CASE as it looks like YELLING!

Next, please try a new installation using the Import option as shown at https://docs.securityonion.net/en/2.4/first-time-users.html. This has lower hardware requirements so if it works correctly then your problems may have been resource related. If that's the case, you can then try doing an Eval installation with higher hardware specs.
